docs/src/examples/pig-breeding.md

```diff
@@ -155,45 +155,19 @@ In Decision Programming:
+## Generating the model
 
-In the pig breeding problem, when the $N$ is large some of the path utilities become negative. In this case, we choose to use the [positive path utility](../decision-programming/decision-model.md) transformation, which allows us to exclude the probability cut in the next section.
+After adding nodes, generating arcs and defining probability and utility values, we generate the model, which is done by `generate_model` function. In this example, we use the slower path-based formulation (model_type="DP").
 
-```julia
-generate_diagram!(diagram, positive_path_utility = true)
-```
-
-## Decision model
-
-Next we initialise the JuMP model and add the decision variables. DecisionVariables has an optional argument names, which will name variables according to node names with state indices if set as true and just as simple indices if set as false. The latter might bring some performance improvements for very large models. The default value is true, which is generally preferable due to more clear naming of variables. 
-
-Then we add the path compatibility variables. Since we applied an affine transformation to the utility function, making all path utilities positive, the probability cut can be excluded from the model. The purpose of this is discussed in the [theoretical section](../decision-programming/decision-model.md) of this documentation.
+By default `generate_model` and `generate_diagram!` use the default path probabilities and utilities, which are defined as the joint probability of all chance events in the diagram and the sum of utilities in value nodes, respectively. In the [Contingent Portfolio Programming](contingent-portfolio-programming.md) example, we show how to use a user-defined custom path utility function. In the pig breeding problem, when the $N$ is large some of the path utilities become negative. In this case, we choose to use the [positive path utility](../decision-programming/decision-model.md) transformation, which allows us to exclude the probability cut in the next section.
 
-```julia
-model = Model()
-z = DecisionVariables(model, diagram, names=true)
-x_s = PathCompatibilityVariables(model, diagram, z, probability_cut = false)
-```
-
-We create the objective function
+Since we applied an affine transformation to the utility function, making all path utilities positive, the probability cut can be excluded from the model (which is also the default setting as). The purpose of this is discussed in the [theoretical section](../decision-programming/decision-model.md) of this documentation.
```

Comment:
typo in the brackets. I don't think "as" belongs there

Reply:
Fixed